src/index.js

```diff
@@ -74,12 +75,25 @@ class KadDHT extends EventEmitter {
      */
     this.kBucketSize = options.kBucketSize || c.K
 
+    /**
+     * ALPHA concurrency at which each query path with run, defaults to 3
+     * @type {number}
+     */
+    this.concurrency = options.concurrency || c.ALPHA
+
+    /**
+     * Number of disjoint query paths to use
+     * This is set to `kBucketSize`/2 per the S/Kademlia paper
+     * @type {number}
+     */
+    this.disjointPaths = Math.ceil(this.kBucketSize / 2)
+
     /**
      * Number of closest peers to return on kBucket search, default 20
      *
      * @type {number}
      */
-    this.ncp = options.ncp || c.K
+    this.ncp = options.ncp || this.kBucketSize
```

Comment:
Can you add docs to `options.ncp` ?

Reply:
So, looking through the code we actually only use `dht.ncp` in 1 spot. I kind of think we should just get rid of it altogether and replace that occurrence with `dht.kBucketSize`. While it could potentially provide some finer grained control of the number of results we're returning, it's not doing that and I think it adds more confusion than it's worth right now. Thoughts?

Reply:
Agree 👍

Reply:
Sounds good!

Reply:
ncp is now gone in favor of `kBucketSize`